deploy/cr.yaml

```diff
@@ -561,7 +561,7 @@ spec:
         cpu: 300m
   backup:
 #    allowParallel: true
-    image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+    image: percona/percona-xtradb-cluster-operator:1.14.0-pxc8.0-backup-pxb8.0.32
```

Comment:
In PXC distribution we have XtraBackup `percona-xtrabackup-80-8.0.35-30.1.el9.x86_64.rpm` so we need to set the same version here so it should be `percona/percona-xtradb-cluster-operator:1.14.0-pxc8.0-backup-pxb8.0.35`

Reply:
done